.circleci/config.yml

```diff
@@ -42,19 +57,71 @@ jobs:
 
       - run: mvn dependency:go-offline
 
+      - run:
+          name: install Redis
+          command: |
+            sudo apt-get -y update
+            sudo apt-get install -y gcc-8 g++-8 libssl-dev
+            wget http://download.redis.io/releases/redis-6.0.10.tar.gz
+            tar -xzvf redis-6.0.10.tar.gz
+            make -C redis-6.0.10 -j`nproc` BUILD_TLS=yes
+            export PATH=$PWD/redis-6.0.10/src:$PATH
+
+      - run:
+          name: Run Test
+          command: |
+            export PATH=$PWD/redis-6.0.10/src:$PATH
+            make test
+
+      - run:
```

Comment:
should we remove this block at all?

Reply:
I'd leave it for now. In the future, I'll move redis to a docker, like other builds.